src/eradiate/experiments/_onedim.py

```diff
@@ -10,12 +10,38 @@
 from ..scenes.atmosphere import Atmosphere, HomogeneousAtmosphere, atmosphere_factory
 from ..scenes.core import KernelDict
 from ..scenes.integrators import Integrator, VolPathIntegrator, integrator_factory
-from ..scenes.measure import Measure, TargetPoint
+from ..scenes.measure import Measure, MultiRadiancemeterMeasure, TargetPoint
 from ..scenes.measure._core import MeasureFlags
 from ..scenes.surface import LambertianSurface, Surface, surface_factory
 from ..units import unit_context_config as ucc
 
 
+def measure_inside_atmosphere(atmosphere, measure, ctx):
+    """
+    Evaluate whether a sensor is placed within an atmosphere.
+
+    Raises a ValueError if calld with a :class:`MultiRadiancemeterMeasure` with
+    origins that are inside as well as outside of the atmosphere.
+    """
+    bbox = atmosphere.eval_bbox(ctx)
+
+    if isinstance(measure, MultiRadiancemeterMeasure):
+        if all(bbox.contains(origin) for origin in measure.origins):
+            return True
+        elif not any(bbox.contains(origin) for origin in measure.origins):
+            return False
```

Comment:
Still with the idea of not evaluating twice:
```suggestion
        bbox_contains = [bbox.contains(origin) for origin in measure.origins]
        if all(bbox_contains):
            return True
        elif not any(bbox_contains):
            return False
```